services/horizon/internal/actions_ledger.go

```diff
@@ -59,6 +59,11 @@ func (action *LedgerIndexAction) SSE(stream sse.Stream) {
 	)
 }
 
+// GetTopic is a method for actions.SSE
+func (action *LedgerIndexAction) GetTopic() string {
+	return "ledger"
```

Comment:
why not `action.GetString(..)` like all other implementations? there are other functions like this in this pr

Reply:
There is no value for ledger there, just "ledger" hardcoded itself

Reply:
There is no value in this action for specific ledger_id, so registration topic is a general change in the ledger.